.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: "🐞Bug report"
+about: Create a report to help us improve (use this to report bugs only)
+title: ''
+labels: "🐞 bug"
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Visit to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots (If Any)**
+Add screenshots to help explain your problem.
+
+**Device Information [optional]:**
```

Comment:
Information that is interesting: 
* Alby version  [e.g. 1.5.0]
* Did you install Alby through the browser stores or manually?
* What wallet did you connect? [e.g. LND, BlueWallet LNDhub]